src/preact/base-element.js

```diff
@@ -43,6 +44,7 @@ import {isLayoutSizeDefined} from '../layout';
  * @typedef {{
  *   attr: (string|undefined),
  *   type: (string|undefined),
+ *   attrPrefix: (string|undefined),
```

Comment:
Please also add to the docs few lines above that explain the "one of" nature of `attr`, `attrs` and now `attrPrefix`.